.gitignore

```diff
@@ -2,3 +2,4 @@
 /Source/SRX.ServiceMonitor/bin
 /Source/SRX.ServiceMonitor/SRX.ServiceMonitor.csproj.user
 /Source/.vs
+/.vs/Source/v17/.suo
```

Comment:
Not sure about why this was checked in, meant to ignore it so it wouldn't get committed in my push, but looks like I didn't do that quite right.